apps/wallet-ui/src/app/globals.css

```diff
@@ -1,70 +1,85 @@
 @tailwind base;
```

Comment:
I'd assume most of this should be 1:1 the same as dashboard?

Reply:
Everything but the primary / secondary colors (we don't want our branding)